service-loadbalancer/service_loadbalancer.go

```diff
@@ -149,6 +150,17 @@ type service struct {
 
 	// Algorithm
 	Algorithm string
+
+	// Indicates if the service must use sticky sessions
+	// http://cbonte.github.io/haproxy-dconv/configuration-1.5.html#stick-table
+	// Enabled using the attribute service.spec.sessionAffinity
+	// https://github.com/kubernetes/kubernetes/blob/master/docs/user-guide/services.md#virtual-ips-and-service-proxies
+	SessionAffinity bool
+
+	// CookieStickySession use a cookie to enable sticky sessions.
+	// The name of the cookie is SERVERID
+	// This only can be used in http services
```

Comment:
Please include a comment like (assuming my comment is accurate, i haven't double checked):
```
If SessionAffinity is set and without CookieStickySession, requests are routed to a backend based on client ip. If both SessionAffinity and CookieStickSession are set, a SERVERID cookie is inserted by the loadbalancer and used to route subsequent requests. If neither is set, requests are routed based on the algorithm.
```

As an aside, what happens when a backend is unhealthy, is it automatically purged from the stick table and all SERVERID cookies are ignored?